src/PKSim.UI.Starter/PKSim.UI.Starter.csproj

```diff
@@ -3,19 +3,28 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-    <Version>11.1.0</Version>
-    <AssemblyVersion>11.1.0</AssemblyVersion>
-    <FileVersion>11.1.0</FileVersion>
+    <Version>12.0.0</Version>
+    <AssemblyVersion>12.0.0</AssemblyVersion>
+    <FileVersion>12.0.0</FileVersion>
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\PKSim.CLI.Core\PKSim.CLI.Core.csproj">
       <CopyLocalSatelliteAssemblies></CopyLocalSatelliteAssemblies>
       <Private></Private>
     </ProjectReference>
+    <ProjectReference Include="..\PKSim.Infrastructure\PKSim.Infrastructure.csproj" />
+    <ProjectReference Include="..\PKSim.Presentation\PKSim.Presentation.csproj" />
     <ProjectReference Include="..\PKSim.UI\PKSim.UI.csproj">
       <Private></Private>
       <CopyLocalSatelliteAssemblies></CopyLocalSatelliteAssemblies>
     </ProjectReference>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
@rwmcintosh @georgeDaskalakis 
How did this ever compile without this?

Reply:
Strange because I've been working in another branch without this ItemGroup and it's been compiling fine.

OSMOSES merged into that branch now - continues to compile.

Reply:
Wondering if this is because you have the stuff locally. At any rate, clean all , rebuild, fail

Reply:
I looked through the Appveyor logs and that project was also building and testing so, I don't know. In fact, the tests from PKSim.UI.Starter are not running locally, but after a commit yesterday the PK-Sim build did not work due to failing tests.